tests/union_duplicates.pyi

```diff
@@ -2,17 +2,16 @@
 import builtins
 import typing
 from collections.abc import Mapping
-from typing import (  # Y022 Use "type[MyClass]" instead of "typing.Type[MyClass]" (PEP 585 syntax)
-    Type,
-    Union,
+from typing import (
+    Type,  # Y022 Use "type[MyClass]" instead of "typing.Type[MyClass]" (PEP 585 syntax)
 )
+from typing import Union
 
 import typing_extensions
-from typing_extensions import (  # Y022 Use "type[MyClass]" instead of "typing_extensions.Type[MyClass]" (PEP 585 syntax)
-    Literal,
-    Type as Type_,
-    TypeAlias,
+from typing_extensions import (
+    Type as Type_,  # Y022 Use "type[MyClass]" instead of "typing_extensions.Type[MyClass]" (PEP 585 syntax
```

Comment:
```suggestion
    Type as Type_,  # Y022 Use "type[MyClass]" instead of "typing_extensions.Type[MyClass]" (PEP 585 syntax)
```
isort deleted the paren at the end of the comment, I'm even less impressed now

Reply:
oh wow

Reply:
OK I fixed this and it put the paren back in the wrong place. I guess we have to either turn off isort or make the error message not end in a parenthesis.